python/tank/platform/qt/fonts/OpenSans/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Yeah, this should go into core's software_credits (as well):
https://github.com/shotgunsoftware/tk-core/blob/master/software_credits

PyYAML and ruamel is there already.